MapboxSearch/offline/src/main/java/com/mapbox/search/offline/NewOfflineSearchResultType.kt

```diff
@@ -0,0 +1,72 @@
+package com.mapbox.search.offline
+
+import androidx.annotation.StringDef
+import com.mapbox.search.base.result.BaseRawResultType
+
+/**
+ * Defines type of the [OfflineSearchResult].
+ * Replaces enum [OfflineSearchResult].
+ */
+public object NewOfflineSearchResultType {
```

Comment:
nit: find a better name, but i guess the old can be removed v3.0 and this one can be renamed in then.

Reply:
Agreed to leave it as is.